core/src/main/java/tc/oc/pgm/stats/TeamStatsInventoryMenuItem.java

```diff
@@ -0,0 +1,149 @@
+                .map(PlayerStatsInventoryMenuItem::new)
+                .collect(Collectors.toList()));
+  }
+
+  @Override
+  public Component getName() {
+    return TranslatableComponent.of("match.stats.team", team.getName());
+  }
+
+  @Override
+  public ChatColor getColor() {
+    return ChatColor.valueOf(team.getColor().name());
+  }
+
+  @Override
+  public List<String> getLore(MatchPlayer player) {
+
+    StatsMatchModule smm = player.getMatch().needModule(StatsMatchModule.class);
+    List<String> lore = new ArrayList<>();
+    int teamKills = 0;
+    int teamDeaths = 0;
+    double damageDone = 0;
+    double damageTaken = 0;
+    double bowDamage = 0;
+    int shotsTaken = 0;
+    int shotsHit = 0;
+    for (MatchPlayer teamPlayer : team.getPlayers()) {
+      PlayerStats stats = smm.getPlayerStat(teamPlayer.getId());
+      teamKills += stats.getKills();
+      teamDeaths += stats.getDeaths();
+      damageDone += stats.getDamageDone();
+      damageTaken += stats.getDamageTaken();
+      bowDamage += stats.getBowDamage();
+      shotsTaken += stats.getShotsTaken();
+      shotsHit += stats.getShotsHit();
+    }
+
+    double teamKD = teamDeaths == 0 ? teamKills : teamKills / (double) teamDeaths;
+    double teamBowAcc = shotsTaken == 0 ? Double.NaN : shotsHit / (shotsTaken / (double) 100);
+    double teamDamageGiven = damageDone;
+    double bowDamageGiven = bowDamage;
+    double teamDamageTaken = damageTaken;
+
+    Component killLore =
+        TranslatableComponent.of(
+            "match.stats.kills.concise", RESET, numberComponent(teamKills, TextColor.GREEN));
+    Component deathLore =
+        TranslatableComponent.of(
+            "match.stats.deaths.concise", RESET, numberComponent(teamDeaths, TextColor.RED));
+    Component kdLore =
+        TranslatableComponent.of(
+            "match.stats.kd.concise", RESET, numberComponent(teamKD, TextColor.GREEN));
+    Component damageLore =
+        TranslatableComponent.of(
+            "match.stats.damage.concise",
+            RESET,
+            numberComponent(teamDamageGiven, TextColor.GREEN),
+            numberComponent(bowDamageGiven, TextColor.YELLOW),
+            numberComponent(teamDamageTaken, TextColor.RED));
+    Component bowLore =
+        TranslatableComponent.of(
+            "match.stats.bow.concise",
+            RESET,
+            numberComponent(shotsTaken, TextColor.YELLOW),
+            numberComponent(teamBowAcc, TextColor.YELLOW));
+
+    Player bukkit = player.getBukkit();
+
+    lore.add(TextTranslations.translateLegacy(killLore, bukkit));
+    lore.add(TextTranslations.translateLegacy(deathLore, bukkit));
+    lore.add(TextTranslations.translateLegacy(kdLore, bukkit));
+    lore.add(TextTranslations.translateLegacy(damageLore, bukkit));
+    lore.add(TextTranslations.translateLegacy(bowLore, bukkit));
+
+    return lore;
+  }
+
+  @Override
+  public Material getMaterial(MatchPlayer player) {
+    return Material.WOOL;
```

Comment:
Coloured leather helm to match the team picker pls 🙏

Reply:
I would not make this a helmet, people will think it's the team picker. Maybe chestplate or leggings (boots should be used for leaving even tho that's not on this version of pgm)

Reply:
I think leggings look weird, but chest plates is a good idea!